Real_Estate/models/real_estate_property.py

```diff
@@ -0,0 +1,37 @@
+from odoo import fields, models
+
+
+class Properties(models.Model):
+    _name = "real.estate.property"
+    _description = "Real Estate Property Table to store information."
+
+    name = fields.Char('Property Name', required=True)
```

Comment:
```suggestion
    name = fields.Char("Property Name", required=True)
```
Any string that will be displayed for user should have `""` any technical string should be `''`
Apply the same below